configure.ac

```diff
@@ -2934,6 +2936,16 @@ AC_CHECK_TYPE(cpu_set_t,
 #endif
 ])
 
+AC_CHECK_TYPE(cpuset_t,
+   AC_DEFINE(HAVE_CPUSET_T,1,[cpuset_t is defined by the system headers]),,[
+#if HAVE_SYS_PARAM_H
+#include <sys/param.h>
+#endif
+#if HAVE_SYS_CPUSET_H
+#include <sys/cpuset.h>
+#endif
+])
```

Comment:
Despite the "cpuset_t is defined by the system headers" description, this code does not seem to check whether `cpuset_t` is defined. For example, it will define HAVE_CPUSET_T even when there are no system headers and, hence, no cpuset_t is defined.